bin/deploy-steps

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Yeah, absolutely... the only problem is we can't run `drush cim -y` in the actual scaffold project, because we don't have config there :( Got any ideas on how to work around that?

We could always leave the "good" steps commented out and add an explicit setup step to fix them.

Reply:
- [ ] Add an explicit step for configuring deploy-steps.